webapp/app/dataExport/components/variableSelect/variableSelect.less

```diff
@@ -0,0 +1,24 @@
+@import (reference) 'webapp/main/styles/config';
+
+.export-variable-select {
+  overflow-y: hidden;
+  display: grid;
+  grid-template-rows: 50px 29px 1px 1fr;
+
+  .divider {
+    border-bottom: 1px solid @ui-border-light;
+  }
+}
+
+.export-variable-select__header {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
```

Comment:
these css classes are basically the same than country select ones. It might be better to generalize them. I can do it after merging this if you wish so

Reply:
Lets do this after last picker, maybe we can refactor more